src/lang-promql/complete/promql.terms.ts

```diff
@@ -18,66 +18,254 @@ export const binOpTerms = [
 ];
 export const binOpModifierTerms = [{ label: 'on' }, { label: 'ignoring' }, { label: 'group_left' }, { label: 'group_right' }];
 export const functionIdentifierTerms = [
-  { label: 'abs' },
-  { label: 'absent' },
-  { label: 'absent_over_time' },
-  { label: 'avg_over_time' },
-  { label: 'ceil' },
-  { label: 'changes' },
-  { label: 'clamp_max' },
-  { label: 'clamp_min' },
-  { label: 'count_over_time' },
-  { label: 'days_in_month' },
-  { label: 'day_of_month' },
-  { label: 'day_of_week' },
-  { label: 'delta' },
-  { label: 'deriv' },
-  { label: 'exp' },
-  { label: 'floor' },
-  { label: 'histogram_quantile' },
-  { label: 'holt_winters' },
-  { label: 'hour' },
-  { label: 'idelta' },
-  { label: 'increase' },
-  { label: 'irate' },
-  { label: 'label_replace' },
-  { label: 'label_join' },
-  { label: 'ln' },
-  { label: 'log10' },
-  { label: 'log2' },
-  { label: 'max_over_time' },
-  { label: 'min_over_time' },
-  { label: 'minute' },
-  { label: 'month' },
-  { label: 'predict_linear' },
-  { label: 'quantile_over_time' },
-  { label: 'rate' },
-  { label: 'resets' },
-  { label: 'round' },
-  { label: 'scalar' },
-  { label: 'sort' },
-  { label: 'sort_desc' },
-  { label: 'sqrt' },
-  { label: 'stddev_over_time' },
-  { label: 'stdvar_over_time' },
-  { label: 'sum_over_time' },
-  { label: 'time' },
-  { label: 'timestamp' },
-  { label: 'vector' },
-  { label: 'year' },
+  {
+    label: 'abs',
+    info: 'Returns the input vector with all sample values converted to their absolute value.',
```

Comment:
could be nice to add also the field `detail` with the value `function`

Reply:
Fixed https://github.com/prometheus-community/codemirror-promql/pull/55#discussion_r498647747